snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -39,7 +39,7 @@ public void Overview()
         {
             var projectId = _fixture.ProjectId;
 
-            // <*>
+            // <Overview>
```

Comment:
If we ever have a method called "Overview", I think we'll have failed anyway :) But we could always change it when we need to (OverviewPage or whatever). It's not like any code has this built into it - I've just referred to it from index.md.
The snippet name does (currently) have to match a regex of `[\w\.]+` though, which `*` wouldn't.